plugin.py

```diff
@@ -18,16 +18,16 @@ def plugin_info(self):
         plugin_name = ''.join(letter for letter in no_space_plugin_name if letter.isalnum() or letter == '_')
         plugin_description = f'Extracts "{file_format}" files and attaches its low-level data structure as a JSON text to the trace.'
         plugin_info = PluginInfo(
-            id=PluginId(domain='{YOUR_ORGANISATION_DOMAIN}', category='extract', name=plugin_name),
-            version='1.0.0',
+            id=PluginId(domain='nfi', category='extract', name=plugin_name),
+            version='1.3.0',
             description=plugin_description,
-            author=Author('{YOUR_NAME}', '{YOUR_EMAIL_ADDRESS}', '{YOUR_ORGANISATION}'),
+            author=Author('Luuk', 'luuk@luuk.com', 'nfi'),
```

Comment:
Should these changes be reverted? (Also see @remco-nfi's comment from 2 weeks ago.)

Reply:
These should indeed be reverted, this is only for testing purposes. Should not have been committed.